data/test_trackers/tracker_moodbot.json

```diff
@@ -22,7 +22,7 @@
     ]
   },
   "sender_id": "mysender",
-  "latest_action_name": "action_listen",
+  "latest_action": {"action_name": "action_listen"},
```

Comment:
@tabergma could you please takeover compatibility comments?

Reply:
Not sure if we actually need to do something here. When creating a tracker from json we don't care about the latest action. The latest action is inferred by the events. So the change does not matter for us internally. However, if users are reading the json, this might cause errors. The json is, for example, returned via our API.
What would be the best way of handling this? Should we change it back to the old way and add a warning that the format of latest action will change in the future?

Reply:
Ok, we just discussed this, we just need to add back "latest_action_name" and then we should be fine. Will also update the API spec.